scene/2d/tile_map.cpp

```diff
@@ -1700,7 +1827,8 @@ TileMap::TileMap() {
 	bounce = 0;
 	mode = MODE_SQUARE;
 	half_offset = HALF_OFFSET_DISABLED;
-	use_kinematic = false;
```

Comment:
Was this removal intended? I don't see this bool being initialized anymore.

Strangely enough, running `doctool` now gives me:
```diff
diff --git a/doc/classes/TileMap.xml b/doc/classes/TileMap.xml
index ee5205efd..55666b94b 100644
--- a/doc/classes/TileMap.xml
+++ b/doc/classes/TileMap.xml
@@ -286,7 +286,7 @@
                <member name="collision_mask" type="int" setter="set_collision_mask" getter="get_collision_mask" default="1">
                        The collision mask(s) for all colliders in the TileMap.
                </member>
-               <member name="collision_use_kinematic" type="bool" setter="set_collision_use_kinematic" getter="get_collision_use_kinematic" default="true">
+               <member name="collision_use_kinematic" type="bool" setter="set_collision_use_kinematic" getter="get_collision_use_kinematic" default="false">
                        If [code]true[/code], TileMap collisions will be handled as a kinematic body. If [code]false[/code], collisions will be handled as static body.
                </member>
                <member name="collision_use_parent" type="bool" setter="set_collision_use_parent" getter="get_collision_use_parent" default="false">
```

Reply:
No idea, I don't remember touching it though.

Reply:
I fixed it in #30184.